wiki/Skinning_Tutorial/osu!mania/en.md

```diff
@@ -1,106 +1,575 @@
+-   This similarly applies to the other elements, but only when playing with 4 keys.
+-   On the ranking screen, the default hit scores will be used.
+
+### Comboburst (basic)
+
+osu!mania combobursts should be facing towards the right, similar to osu!standard and osu!catch. However, these images are not placed along the bottom adjacent corners of the screen. Thus, osu!mania combobursts should be complete images and not be cropped. All osu!mania combobursts will be displayed on the right side of the stage.
+
+### Comboburst (advanced)
+
+Using `ComboBurstStyle`, you can set which side of the stage the osu!mania combobursts should be displayed on. These should still be facing towards the right, similar to osu!standard and osu!catch combobursts, as osu! will flip these horizontally when appearing on the left side of the stage.
+
+### Mod icons
+
+osu!mania has a few specific mod icons that you can skin:
+
+-   Fade In
+-   xK
+-   Co-Op
+-   Random
+
+On a side note, there was a mod called Fade Out which replaced the Hidden mod icon when osu!mania was the set game mode, but this was removed from the game. You could skin the hidden mod icon to make it appear as Fade Out, but this may look weird when playing in other game modes.
+
+### Shared elements
+
+The following elements are used with other parts of osu!:
+
+-   scorebar and all its files
+-   combo numbers
+-   score numbers
+-   mod icons, except the mode-specific mods
+
+## Playfield design
+
+### Combo numbers
+
+By default, combo numbers are centred horizontally in the stage. The vertical positioning can be changed by setting `ComboPosition`.
+
+If there are two stages, the combo number will be displayed on both stages (this cannot be changed).
+
+### Hit scores
+
+Like combo numbers, hit scores are centred horizontally in the stage. The vertical positioning can be changed by setting `ScorePosition`.
+
+By default, if there are two stages, the hit scores from each stage will be displayed on the both stages. To change this behaviour (have hit scores display on the stage the note was hit on), set `SeparateScore` to `1`.
+
+### Centring the stage
+
+*Notice: Centring of the stage will be inconsistent between differing ratio aspects. You may want to provide extra download links for other common ratio aspects.*
+
+To centre the osu!mania stage, use the following equation to set `ColumnStart`.
+
+`ColumnStart = ((480 * screenWidth / screenHeight) - (columnWidths + ColumnSpacings)) / 2`.
+
+-   `screenWidth` = the game client's width (or the ratio aspect width)
+-   `screenHeight` = the game client's height (or the ratio aspect height)
+-   `columnWidths` = sum of `ColumnWidth`
+-   `ColumnSpacings` = sum of `ColumnSpacing`
+
+*Notice: You will need to round the number to make it an integer. Because of this, the stage may be off by a unit to the left or right. In most cases, this will not be very noticiable.*
```

Comment:
```suggestion
*Notice: You will need to round the number to make it an integer. Because of this, the stage may be off by a unit to the left or right. In most cases, this will not be very noticeable.*
```